runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -2983,7 +2983,7 @@ void chpl_comm_impl_regMemPostAlloc(void* p, size_t size)
             "chpl_comm_regMemPostAlloc(): entry %d, bcast",
             mr_i);
     PERFSTATS_INC(regMem_bCast_cnt);
-    regMemBroadcast(mr, sizeof(*mr));
+    regMemBroadcast(mr_i, 1, false /*send_mreg_cnt*/);
```

Comment:
I tend to prefer `/*send_mreg_cnt=*/false` over `false /*send_mreg_cnt*/`

Reply:
I'm tempted, but in this file there are already other instances of this sort of thing, using the same construction I did here.  So I think in the name of consistency I'll leave it for now.